cluster_pack/filesystem.py

```diff
@@ -191,6 +211,13 @@ def __init__(self, base_fs: Any):
         if isinstance(base_fs, pyarrow.filesystem.LocalFileSystem):
             base_fs.chmod = types.MethodType(_chmod, base_fs)
             base_fs.rm = types.MethodType(_rm, base_fs)
+            base_fs.rename = types.MethodType(_rename, base_fs)
+
+        if isinstance(base_fs, pyarrow.filesystem.S3FSWrapper):
+            base_fs.rename = types.MethodType(_s3_rename, base_fs)

```

Comment:
I don't fully get why we are adding a 'rename' function here, it is not used after. 🤔 
Isn't there any confusion with the `mv` function maybe ?

Reply:
mv is calling rename in the base class